docs/en_US/regions.md

```diff
@@ -0,0 +1,14 @@
+# Unsupported Regions
+
+Pretendo does not support access from certain countries due to technical and legal limitations. These restrictions are primarily due to Cloudflare's network limitations and other compliance factors.
```

Comment:
```suggestion
Pretendo Network does not support access from certain countries due to technical and legal limitations. These restrictions are primarily due to Cloudflare's network limitations and other compliance factors.
```

We always refer to ourselves by our full name in official documents (there are cases of this being missing, but those should also be fixed to include `Network`)